### PR TITLE
Changes regarding 'shortmess', ":h :echo-redraw", and use of sigils

### PR DESCRIPTION
**Pull request by lifecrisis**
Hi @rburny,

I made some minor improvements to your script. I think it's a great feature, and I've had fun using it at work. I actually think this feature should be implemented in the Vim core, perhaps as an option. That's way down the road, though. You'd have to wade through an ocean of C to get it done.

The reasons for my changes are all contained in the commit messages, themselves.

Thanks for your review, and for the plugin!

### CODE REVIEW

**Comment by lifecrisis** [comment]
**Note:** Please approve my changes if you desire, and I will then sign the Google CLA.

File: plugin/searchindex.vim [hunk 1]
```diff
@@ -38,6 +38,8 @@ if !exists('g:searchindex_star_case')
   let g:searchindex_star_case=1
 endif
 
+set shortmess+=s
```
Comment (rburny) [inline plugin/searchindex.vim]: Please add a short comment here, it's easier than looking it up in Vim help every time.

**Comment by rburny** [comment]
Thanks; these look good. Please sign the CLA, and I'll merge your changes.

**Comment by lifecrisis** [comment]
Hey @rburny, I've signed the CLA using my Github username. Should be all good to go!

**Comment by rburny** [comment]
Thank you!

**Pull request merged**